clang/test/CheckedCRewriter/definedType.c

```diff
@@ -110,3 +110,37 @@ baz (*(*lua_test3));
 typedef int *StkId;
 void lua_test4(StkId *x) {}
 // CHECK: void lua_test4(_Ptr<_Ptr<int>> x) _Checked {}
+
+// Things declared inside macros should be WILD unless we start doing something extremely clever
+
+#define declare_function(x) int *foo##x(int *a) {return a; } int *bar##x(int *a) { return a; }
```

Comment:
What about a case where concatenation is not involved? E.g. 
```
#define foo(x) int *x = 0; 
int *bar(void) {
  foo(z)
  foo(y)
  return z;
}
```

Reply:
No conversion happens.

Reply:
Right, I'm saying that perhaps this should be added as a test. The other tests you have seem to all use concatenation `##` and are at the function-def level, not within a function. Maybe that doesn't matter but I thought we should test it?

Reply:
Shouldn't be relevant, but not harm in adding a test.